tools/components/admin/requests/forms/types.ts

```diff
@@ -57,3 +57,28 @@ export type DoctorInformation = Pick<
   Physician,
   'mspNumber' | 'name' | 'phone' | 'addressLine1' | 'addressLine2' | 'city' | 'postalCode'
 >;
+
+// Physician Assessment Information Object
+export type PhysicianAssessmentInformation = Pick<
+  Application,
+  'disability' | 'affectsMobility' | 'mobilityAidRequired' | 'cannotWalk100m' | 'permitType'
+> & {
+  physicianCertificationDate: string;
+  patientEligibilityDescription?: string;
+  temporaryPermitExpiryDate?: string;
```

Comment:
These need DB changes. Could you please add a TODO on top so we know to account for these columns?